rosidl_generator_dotnet/cmake/rosidl_generator_dotnet_generate_interfaces.cmake

```diff
@@ -35,6 +35,15 @@ set(_output_path
 set(_generated_cs_files "")
 set(_generated_c_ts_files "")
 set(_generated_h_files "")
+set(ros2_distro "$ENV{ROS_DISTRO}")
+set(humble_distro_and_newer humble iron rolling)
```

Comment:
I think we should invert the logic here:
This list would need to be extend going forward, but if we have a list of older ROS2 distros (I think `foxy` and `galacic` should be enougth) this won't break if j-turtle gets released.

Reply:
I inverted the logic on the last commit